pkg/roachpb/api.proto

```diff
@@ -943,6 +943,7 @@ message AdminScatterRequest {
   option (gogoproto.equal) = true;
 
   optional Span header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
+  optional bool disableStatsBasedRebalance = 2 [(gogoproto.nullable) = false];
```

Comment:
Is there a reason the API change is needed? The implementation of `AdminScatter` can just always disable stats-based rebalancing internally.

Reply:
Hmm, You are right. My mistake.